roles/cloud_connector/tasks/cleanup_receptor.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Find receptor configs
+  find:
+    paths: /etc/receptor
+    file_type: directory
+  register: receptor_configs
+
+- name: Identify Satellite ID from Satellite
```

Comment:
does this step have to happen before `delete_source.yaml`? It feels wrong to "find receptor configs" first, then get the ID which is totally unrelated, and then clean up stuff for receptor (again).

Reply:
ah, reading the source of `delete_source.yaml` explains it. then i'd order this: get id, find configs, include delete?